packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.ts

```diff
@@ -169,15 +177,119 @@ describe("End to end chunked encoding", () => {
 			fieldBatchCodec,
 			context,
 			options,
+			idCompressor,
 		);
 
 		// This function is declared in the test to have access to the original uniform chunk for comparison.
 		function stringifier(content: unknown) {
-			const insertedChunk = decode((content as any).fields);
+			const insertedChunk = decode((content as any).fields, idCompressor);
 			assert.equal(insertedChunk, chunk);
 			assert(chunk.isShared());
 			return JSON.stringify(content);
 		}
 		forestSummarizer.getAttachSummary(stringifier);
 	});
+	describe("identifier field encoding", () => {
+		it("is encoded as compressed id when the identifier is a valid stable id.", () => {
+			const schema = new SchemaFactory("com.example");
```

Comment:
these two tests can probably share a lot of code (schema types, codec options, etc.)

Reply:
Refactored tests and created `getIdentifierEncodingContext` function to reuse repeated code